### PR TITLE
Game Screen Over

### PR DESCRIPTION
**Pull request by ChrisL119**
## Changes
_List Changes Introduced by this PR_
1Added Game Over Screen

## Purpose
-When the timer reaches 0 a game over screen should pop up
-Use the Quit function from the Paws menu to restart

### CODE REVIEW

**Review by Tabspear — APPROVED** [review]
Awesome job :)

**Review by gabymoreno9 — CHANGES_REQUESTED** [review]
so the cat goes back into position which like is half the battle but theres no game over popup

**Comment by gabymoreno9** [comment]
What you can do for now: put in a placeholder HTML element overlay. For now wait for the pause menu to be done.

File: js/script.js [hunk 1]
```diff
@@ -333,19 +333,20 @@ start_button.addEventListener("click", function () {
...
-    clearInterval(i);
-    if (i === 0) {
-      alert("Game Over!");
-    }
-  } else {
-    setTimeout(onTimer, 1000);
+  timeout = setTimeout(onTimer, 1000);
+  if (i <= stopTime) {
+    clearTimeout(timeout);
+    setTimeout(() => window.open('http://127.0.0.1:5505/game%20over.html'), 1000);
```
Comment (itannady) [inline js/script.js:346]: Is this a placeholder page? 
Screen Shot 2022-10-12 at 4 28 52 PM

**Review by itannady — COMMENTED** [review]
Timer function runs great! It redirects to a game over page, but it's currently blank on my end. Is this a placeholder page?

File: js/script.js (hunk 1 above)
Comment (ChrisL119) [reply js/script.js:346]: no it should be working but I barely realized i git stashed the game over HTML fixing that right now

**Review by Big3D — APPROVED** [review]

**Pull request merged**